ul/src/association/client.rs

```diff
@@ -251,6 +251,8 @@ impl<'a> ClientAssociationOptions<'a> {
                 protocol_version: protocol_version_scp,
                 application_context_name: _,
                 presentation_contexts: presentation_contexts_scp,
+                calling_ae_title,
+                called_ae_title,
```

Comment:
When not used, we can unbind them.
```suggestion
                calling_ae_title: _,
                called_ae_title: _,
```

Reply:
Nice...